perf_sim/main.cpp

```diff
@@ -1,8 +1,6 @@
 /**
- * main.cpp
- * Start point of the scalar MIPS CPU simulator.
- * MIPT-MIPS Assignment 4.
- * Ladin Oleg.
+ * main.cpp - entry point of the scalar MIPS CPU simulator
+ * Copyright 2017 MIPT-MIPS
```

Comment:
The purpose of keeping author's name is to left a contact person's name if there are some problems. Usually authorship is tracked by CVS, but history may be unavailable in some cases.
So, you have either to add your name or to keep the old one.